src/core/AutoRest/Simplify/CSharpSimplifier.cs

```diff
@@ -92,9 +92,20 @@ private static MetadataReference Mscorlib
                         // get rid of any BOMs 
                         .Trim('\x00EF', '\x00BB', '\x00BF', '\uFEFF', '\u200B');
 
-                    // Write out the files back to their original location
-                    var output = Path.Combine(Settings.Instance.FileSystem.CurrentDirectory, document.Name);
-                    Settings.Instance.FileSystem.WriteFile(output, text);
+
+                    // special cases the simplifier can't handle.
+                    text = text.
```

Comment:
and we're sure that no one ever says `default: "[Newtonsoft.Json.JsonObject]"` or `enum: ["Newtonsoft.Json.JsonObject]", "cowbell"]` which would end up in the source code and then get replaced here? ⚡️